modules/virt-add-disk-to-vm.adoc

```diff
@@ -27,8 +27,8 @@ Use this procedure to add a virtual disk to a {object}.
 .Procedure
 
 . Click *Workloads* -> *Virtualization* from the side menu.
-. Click the *{object-gui}s* tab.
-. Select a {object} to open the *{object-gui} Overview* screen.
+. Click the *{object-gui}* tab.
+. Select a {object} to open the *VM Template Details* pane.
```

Comment:
Changing this from the variable to "VM Template Details" will mean that it will build as "VM Template Details" in both the VM assembly and the Template assembly, which you probably don't want (unless it's the same across both, in which case you can ignore me).
If you can't find a way to write around it, you could declare another variable (for both VM and Template, where the conditionals are established) and use that instead.

Reply:
@aburdenthehand - will look into this after QE review.